views/view-elasticsearch-hosts.php

```diff
@@ -74,13 +74,13 @@ function bool2str($v) {
             if ($shard['state'] != 'RELOCATING') {
                 unset($shard['relocating_node']);
             }
-            foreach ($shard as $k => $v) {
+            foreach ($shard as $k => $variable) {
```

Comment:
`$k` and `$v` here represent **key** and **value**, if you are going expand them it would be best to use the correct words :grin: